### ISSUE TITLE
Constant val_loss (even with training set as dev)

### ISSUE THREAD

**Issue by enavarroAI** [keras-team/keras#8467]
Hello,
I have been using Keras for a while now, but I am having an issue that I cannot seem to be able to solve.
I am training an InceptionV3 CNN, without the default classification block. I create the classification network in top of the CNN. All correct til here.
Then, I train the network on images (139,139,3), the minimum size permited according to the InceptionV3 specifications in the documentation. The training loss decreases, so it means it is working. However, the validation loss is ALWAYS the same value, no matter that the train loss decreases. To debug it, I put the training data as validation data, and the same happens.
I know that there is no Dropout and validation time, but still, the behaviour is not as expected (and also I removed the Dropout layers in other tests, and the same happened). I tried both in Theano and Tensorflow.

Any idea what is happening? My guess is that the model is always predicting one of the two classes when evaluating the dev set, even tho it learns how to separate them while training. Might be a problem with the implementation of the fit method, in terms of predict instead of predict_proba?
Thank you!!

PS: Sometimes, the val loss goes down, but then goes up again and gets stuck (1/20 times)

```python
from keras.applications.inception_v3 import InceptionV3
base_model = InceptionV3(include_top=False, weights=None, input_tensor=None, input_shape=(w,h,3), pooling=None, classes=1)
x = Flatten()(base_model.output)
x = Dense(256, activation='relu')(x)
x = Dropout(0.5)(x)
x = BatchNormalization()(x)
x = Dense(64, activation='relu')(x)
x = Dropout(0.5)(x)
predictions = Dense(1, activation = 'sigmoid')(x)
head_model = Model(base_model.input, predictions)
head_model.compile(optimizer='adam', loss='binary_crossentropy')
model_history = head_model.fit(X_train, Y_train, validation_data = (X_train,Y_train), 
                          batch_size = 64, epochs = 100, 
                          verbose = 1)

Train on 1604 samples, validate on 1604 samples
Epoch 1/100
1604/1604 [==============================] - 292s - loss: 0.6994 - val_loss: 7.3098
Epoch 2/100
1604/1604 [==============================] - 290s - loss: 0.5783 - val_loss: 7.3098
Epoch 3/100
1604/1604 [==============================] - 293s - loss: 0.5301 - val_loss: 7.3098
Epoch 4/100
1604/1604 [==============================] - 291s - loss: 0.4503 - val_loss: 7.3098
```

**Comment by coldfir3** [comment]
Sometimes i have the exact same problem, have you fixed it?
